PlanetExpress/src/PlanetExpress/Crew.java

```diff
@@ -0,0 +1,118 @@
+package PlanetExpress;
+
+public class Crew {
+	
+	public enum Action {
+		DRINK, DELIVER, STEAL, EAT, ACCOUNT
+	}
+
+	Member[] members;
```

Comment:
Seems like this array isn't really necessary, but it's nice that each of the "people" are members in case you did want to do a single operation on then all